exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Table.kt

```diff
@@ -370,6 +370,7 @@ open class Table(name: String = "") : ColumnSet(), DdlAware {
         init {
             checkMultipleDeclaration()
             for (column in columns) column.markPrimaryKey()
+            columns.sortWith(compareBy { !it.columnType.isAutoInc })
```

Comment:
Why is this neccessary?

Reply:
Make auto inc columns in first order while building the primary key contraint. 
Exemple of test case :
````kotlin
@Test fun tableWithMultiPKandAutoIncrement() {
        val Foo = object : IdTable<Long>("FooTable") {
            val bar = integer("bar")
            override val id: Column<EntityID<Long>> = long("id").entityId().autoIncrement()

            override val primaryKey = PrimaryKey(bar, id)
        }
````
We need to have id before bar : `CONSTRAINT pk_FooTable PRIMARY KEY (ID, BAR)`

We already doing the same thing in the old method :
````kotlin
    private fun getPrimaryKeyColumns(): List<Column<*>>? = columns
        .filter { it.indexInPK != null }
        .sortedWith(compareBy({ !it.columnType.isAutoInc }, { it.indexInPK }))
        .takeIf { it.isNotEmpty() }
````